### PR TITLE
Add ability to set subscriber answers to select questions

### PR DESCRIPTION
**Pull request by Scotchester**
For the new conference registration functionality update, we need the ability to set subscribers' answers to **select** questions (i.e., questions that have set answers to choose from, as opposed to free response questions). This PR adds a method to do that.

### CODE REVIEW

**Comment by Scotchester** [comment]
Here's the GovDelivery API docs for the call this is making:

https://developer.govdelivery.com/api/comm_cloud_v1/Default.htm#API/Comm%20Cloud%20V1/API_CommCloudV1_Subscribers_ChangeSingleResponse.htm%3FTocPath%3DCommunications%2520Cloud%2520API%2520v1|Resources|Subscribers|Subscriber%2520Question%2520Responses|_____4

File: govdelivery/api.py [hunk 1]
```diff
@@ -130,3 +130,29 @@ def set_subscriber_answers_to_question(self, contact_details, question_id, answe
...
+        contact_details,
+        question_id,
+        answer_id
+    ):
+        subscriber_id = base64.b64encode(contact_details)
+        question_id_encoded = base64.b64encode(question_id)
+        answer_id_encoded = base64.b64encode(answer_id)
+
+        path = self.translate_path(
+            '/api/account/$account_code/subscribers/$subscriber_id/questions/'
```
Comment (chosak) [inline govdelivery/api.py]: Minor style thing: at first read, while valid, the breaking of the string on two lines like this is unexpected compared with the other arguments to the method call. I might more clearly separate this out like

```py
path = self.translate_path(
    (
        '/api/account/$account_code'
        '/subscribers/$subscriber_id'
        '/questions/$question_id_encoded/responses.xml'
    ),
    subscriber_id=subscriber_id,
    ...
)

File: govdelivery/api.py [hunk 2]
```diff
@@ -130,3 +130,29 @@ def set_subscriber_answers_to_question(self, contact_details, question_id, answe
 
         payload = xml_payloads.free_response_to_question(question_id_encoded, answer_text)
         return self.call_api(path, "put", payload)
+
+    set_subscriber_answer_to_free_response_question = set_subscriber_answers_to_question
```
Comment (chosak) [inline govdelivery/api.py]: I see why you want to make this change; I'm not sure I would do it this way without maybe also including [some kind of deprecation warning](https://stackoverflow.com/questions/2536307/decorators-in-the-python-standard-lib-deprecated-specifically/30253848#30253848). There aren't many other uses of this library on public GitHub, but [there are a few](https://github.com/search?q=set_subscriber_answers_to_question&type=Code).

I might revert this particular change and handle it as a separate PR, especially given the lack of tests on this repo. If the name of the previous method is confusing, we could also add a docstring.

**Review by chosak — APPROVED** [review]

File: govdelivery/api.py (hunk 1 above)
Comment (Scotchester) [reply govdelivery/api.py]: I took it one step further, hope that's OK :)

File: govdelivery/api.py (hunk 2 above)
Comment (Scotchester) [reply govdelivery/api.py]: Removed for a future PR.

**Pull request merged**